apps/studio/src/features/editing-experience/components/EditPageDrawer.tsx

```diff
@@ -0,0 +1,53 @@
+import { Text, VStack, HStack, Box, Divider, Spacer } from '@chakra-ui/react'
+import { useState } from 'react'
+import {
+  DragDropContext,
+  Droppable,
+  Draggable,
+  DropResult,
+} from '@hello-pangea/dnd'
+import { MdOutlineDragIndicator } from 'react-icons/md'
+import { BsPlus } from 'react-icons/bs'
+import { Button } from '@opengovsg/design-system-react'
+import RootStateDrawer, { Block } from './RootStateDrawer'
+
+export type RootDrawerState = {
+  state: 'root'
+  blocks: Block[]
+}
+
+type AddNewBlockState = {
```

Comment:
nit: can just use the string as the type 
```ts
type AddNewBlockState = "add-new-block"
```

not a big diff but we don't have to call `.state` to access

Reply:
ah intentionally left it like this for future extension